code/modules/clothing/suits/armor.dm

```diff
@@ -86,8 +86,8 @@
 	max_upgrades = 1
 	armor_list = list(
 		melee = 10,
-		bullet = 11,
-		energy = 10,
+		bullet = 12,
+		energy = 5,
```

Comment:
```suggestion
		energy = 7,
```
5 is like really really bad for the kinda pricey armor we are talking here. please do a 7 like I suggested in the original PR? 7 is like what the adv paramedic/nanite RIG gets. It aint that much. But enough to not be worthless.